tinymce-basics/customizing-the-table-toolbar.html

```diff
@@ -0,0 +1,583 @@
+                });
+
+                editor.addShortcut('alt+40', "Insert row after", () => {
+                    menuActionManager.insertRowAfter();
+                });
+
+                editor.addShortcut('ctrl+meta+M', "Merge selected cells", () => {
+                    menuActionManager.mergeCells();
+                });
+
+                editor.addShortcut('ctrl+shift+meta+M', "Split cell", () => {
+                    menuActionManager.splitCell();
+                });
+
+                // TinyMCE 5.4 introduced improved APIs for inserting tables.
+                // Here we create a toolbar button that immediately inserts
+                // a 3x2 table with a header into the content.
+                // https://www.tiny.cloud/docs/ui-components/toolbarbuttons/
+                editor.ui.registry.addButton('inserttable', {
+                    icon: 'table',
+                    tooltip: 'inserttable',
+                    onAction: () => {
+                        // https://www.tiny.cloud/docs/plugins/table/#commands
+                        editor.execCommand('mceInsertTable', false, { rows: 3, columns: 2, options: { headerRows: 1 } });
+                    }
+                });
+
+                // Create a toolbar menu button to hold all row related options
+                // https://www.tiny.cloud/docs/ui-components/typesoftoolbarbuttons/#menubutton
+                editor.ui.registry.addMenuButton('rowoptions', {
+                    icon: 'table-row-properties', // Part of the default icon pack: https://www.tiny.cloud/docs/advanced/editor-icon-identifiers/
+                    tooltip: 'Row options',
+                    fetch: (callback) => {
+                        var items = [
+                            {
+                                type: 'togglemenuitem',
+                                text: 'Header row',
+                                icon: 'table-top-header',
+                                onAction: () => {
+                                    menuActionManager.headerRow();
+                                },
+                                onSetup: (buttonApi) => {
+                                    // Set the on/off state of the toggle button by query the type and return a boolean
+                                    const state = editor.queryCommandValue('mceTableRowType') === 'body' ? false : true;
+                                    buttonApi.setActive(state);
+                                }
+                            },
+                            {
+                                type: 'separator',
+                            },
+                            {
+                                type: 'menuitem',
+                                text: 'Insert row before',
+                                icon: 'table-insert-row-above',
+                                shortcut: 'meta+alt+↑', // Text displayed in the menuitem's shortcut label
```

Comment:
I think there's a typo here, as just doesn't match the shortcut being registered, as above it's registered as `alt+↑`.

Reply:
All the little details... Thanks for catching this and the other ones